### PR TITLE
Publish security advisories for 3.6.1 and 2.28.9

### PR DESCRIPTION
**Pull request by davidhorstmann-arm**
Publish security advisories approved previously in private.

### CODE REVIEW

File: security-advisories/mbedtls-security-advisory-2024-08-1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# CTR\_DRBG prioritized over HMAC\_DRBG as the PSA DRBG
+
+**Title** | CTR\_DRBG prioritized over HMAC\_DRBG as the PSA DRBG
+--------- | -----------------------------------------------------
+**CVE** | TODO
+**Date** | TODO
```
Comment (yanesca) [inline security-advisories/mbedtls-security-advisory-2024-08-1.md:6]: We might be still waiting for the CVE, but I think we know the date now. (Same for the other two.)

File: security-advisories/mbedtls-security-advisory-2024-08-1.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# CTR\_DRBG prioritized over HMAC\_DRBG as the PSA DRBG
+
+**Title** | CTR\_DRBG prioritized over HMAC\_DRBG as the PSA DRBG
+--------- | -----------------------------------------------------
+**CVE** | TODO
```
Comment (gilles-peskine-arm) [inline security-advisories/mbedtls-security-advisory-2024-08-1.md:5]: Please fill in the TODOs

**Review by yanesca — DISMISSED** [review]
I have confirmed that the content is the same as approved in restricted. However, now we know the release date and shouldn't leave that as a TODO. I am not sure if we have the CVEs yet, but if we don't we should raise an issue about it to make sure that we don't forget.

File: security-advisories/mbedtls-security-advisory-2024-08-1.md (hunk 1 above)
Comment (gilles-peskine-arm) [reply security-advisories/mbedtls-security-advisory-2024-08-1.md:6]: The CVE IDs are in the changelog.

Comment (davidhorstmann-arm) [reply security-advisories/mbedtls-security-advisory-2024-08-1.md]: Apologies, I forgot that there were updates to do. I have filled in both CVE and date.

**Pull request merged**